### DISCUSSION TITLE
Should `role` be included in global attributes?

### DISCUSSION THREAD

**Discussion by estelle** [mdn/mdn-community#186] [Content] [answered]
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes

There is an entire section devoted to the role attribute -- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles -- but on the global attributes page, the only content is "The ARIA [role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles) attribute and the multiple [aria-*](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes) states and properties, used for ensuring accessibility."

Should we add a section on role between `part` and `slot`, or should we not b/c role is defined in WAI-ARIA and not in WHATWG?

**Comment by teoli2003** [comment, marked as answer]
Yes! I think that in which spec they are defined is not important: if I find an attribute prefixed with `aria-`, I look in the `aria-` (global) attributes. If not, I look at the element, then at the global attribute list.

I would definitively love to see such a section added! That page is where I looked at it first (yesterday).

**Reply by estelle** [reply]
https://github.com/mdn/content/issues/19942
